lms/templates/login.html

```diff
@@ -315,3 +314,14 @@ <h2 class="sr">${_('Required Information')}</h2>
   	</div> <!-- div tab-content -->
   </main>
 </div> <!-- div1 -->
+%if banner_account_activation_message:
+<script>
+    let $email = $('#email');
+    $email.blur();
```

Comment:
Jak dla mnie to powinnaś zapewnić aby, ta funkcja wywoływała się gdy już wczyta się cały html
$(function() { ... });

Reply:
zmienione